### PR TITLE
Changed 'conText' key for constitution tooltip to 'constText'.

### PR DESCRIPTION
**Pull request by ghost**
Addresses HabitRPG/habitrpg#2572.

### CODE REVIEW

File: locales/en/main.json [hunk 1]
```diff
@@ -118,7 +118,7 @@
       "strength" : "Strength",
         "strengthText" : "Strength reduces task threat (redness), increases the Gold and Experience boost from random \"critical hits\", and helps deal damage to boss monsters.",
       "constitution" : "Constitution",
-        "conText" : "Constitution reduces the damage you take, whether from negative Habits, missed Dailies, or the attacks of boss monsters.1",
+        "constText" : "Constitution reduces the damage you take, whether from negative Habits, missed Dailies, or the attacks of boss monsters.1",
```
Comment (colegleason) [inline locales/en/main.json]: Can you remove the 1 at the end of this line while you are at it please?

**Comment by marcelosizer** [comment]
@ryepdx I just finished splitting main.json file and I altered this var here [habitrpg #2583](https://github.com/HabitRPG/habitrpg/pull/2583) and here #119. This PR would cause version conflict. Sorry about that. =/

**Comment by lefnire** [comment]
shoot, crap... sorry @Sinza- didn't see that note. I think @paglias said somewhere he was gonna do some merging tomorrow, so I'll leave it to him - but @paglias if you need me to do anything instead, let me know.

@Sinza- I added you as a repo admin btw, feel free to directly commit hotfixes / typos, etc. Or things you've tested & know work. Keep sending the rest as PRs. Thanks for all your help man!

**Comment by marcelosizer** [comment]
@lefnire ok, will do. =]

I think this won't break anything, the worst thing that can happen is that it will revert back to the previous version when @paglias merges the main.json split as I left this one intact.

**Comment by colegleason** [comment]
@ryepdx What is your UUID for a contrib.lvl++ (unless @lefnire already got you)?

**Comment by ghost** [comment]
d12794b4-9d71-4dcd-a3d7-ad0f62064091

Thanks!

**Pull request merged**